beacon-chain/usage_test.go

```diff
@@ -15,24 +16,26 @@ func TestAllFlagsExistInHelp(t *testing.T) {
 	for _, group := range appHelpFlagGroups {
 		helpFlags = append(helpFlags, group.Flags...)
 	}
+	helpFlags = featureconfig.ActiveFlags(helpFlags)
+	appFlags = featureconfig.ActiveFlags(appFlags)
 
 	for _, flag := range appFlags {
 		if !doesFlagExist(flag, helpFlags) {
-			t.Errorf("Flag %s does not exist in help/usage flags.", flag.GetName())
+			t.Errorf("Flag %s does not exist in help/usage flags.", flag.Names()[0])
```

Comment:
Any reason to use this construction rather than `flag.Name`?

Reply:
Sorry, I didn't see this. The flag interface doesn't have flag.Name in v2. https://pkg.go.dev/gopkg.in/urfave/cli.v2?tab=doc#Flag